dandiapi/api/tests/test_stats.py

```diff
@@ -1,15 +1,18 @@
 from __future__ import annotations
 
+from django.contrib.auth.models import User
 import pytest
 
+from dandiapi.api.models import UserMetadata
+
 
 @pytest.mark.django_db()
 def test_stats_baseline(api_client):
     assert api_client.get('/api/stats/').data == {
         'dandiset_count': 0,
         'published_dandiset_count': 0,
-        # django-guardian automatically creates an AnonymousUser
-        'user_count': 1,
+        # django-guardian automatically creates an AnonymousUser, but user is not approved
```

Comment:
```suggestion
```

I don't think the comment is needed any more.

Reply:
Updated